src/lib/utilities/route-for-api.ts

```diff
@@ -23,6 +25,7 @@ type APIRouteParameters = {
   queue: string;
 };
 
+export type ArchiveRouteParameters = Pick<APIRouteParameters, 'namespace'>;
```

Comment:
Do we need this type or can we just use `WorkflowListRouteParameters` since they're the same.